### ISSUE TITLE
Does it only work on Minecraft v.1.8.8?

### ISSUE THREAD

**Issue by robert-dzikowski** [PistonDevelopers/hematite#314]
I try tried to run it using Minecraft 1.19.4., but I got error
```
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', src\minecraft\block_state.rs:285:14
```

**Comment by Caellian** [comment]
Yes, it seems hematite currently doesn't handle blocks in the world file which are missing from its [data (`src/minecraft/data_1_8_pre2.rs`)](https://github.com/PistonDevelopers/hematite/blob/1fc08c93ad10f948a9b74cdfca0915ce009addb7/src/minecraft/data_1_8_pre2.rs).
